### DISCUSSION TITLE
I want to localize

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#258] [Q&A]
What files should I modify if I want to localize the UI?

**Comment by danny-avila** [comment]
I'm not sure what you mean by localizing the UI

**Comment by fuegovic** [comment]
Are you referring to localizing as in translating the UI elements into different languages?

**Reply by tttll1091** [reply]
yes

**Reply by fuegovic** [reply]
Sadly there are no simple way to do it at the moment. The easiest method would be to use the browser automatic translation features for now.

**Reply by tttll1091** [reply]
Thanks
